backend/app/load_data.py

```
@@ -125,6 +126,42 @@ def populate_book_names_table(db_: Session, file: str) -> None:
+    try:
+        record = db_models.VersificationSchema(
+            name="English-ERV",
+            description=None,
+            data=data,
+            is_default=True,
+        )
+
+        db_.add(record)
+        db_.commit()
+
+    except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)